examples/causality/plot_inter_and_intra_lobe_causality.py

```diff
@@ -0,0 +1,79 @@
+"""
+Group a causality matrix by lobes and plot the resulting
+inter- and intra-lobe causality.
+
+Author: Christian Kiefer <ch.kiefer@fz-juelich.de>
+"""
+
+import os
+import os.path as op
+
+import matplotlib.pyplot as plt
+import mne
+import numpy as np
+
+from jumeg.connectivity.con_utils import group_con_matrix_by_lobe
+from jumeg.connectivity.con_viz import plot_grouped_causality_circle
+from jumeg.jumeg_utils import get_jumeg_path
+
+###############################################################################
+# Load the grouping files
+###############################################################################
+
+grouping_yaml_fname = op.join(get_jumeg_path(), 'data',
+                              'desikan_aparc_cortex_based_grouping_ck.yaml')
+lobe_grouping_yaml_fname = op.join(get_jumeg_path(), 'data',
+                                   'lobes_grouping.yaml')
+
+###############################################################################
+# Load anatomical labels
+###############################################################################
+
+subjects_dir = os.environ['SUBJECTS_DIR']
+
+full_labels = mne.read_labels_from_annot(subject='fsaverage', parc='aparc',
+                                         hemi='both', subjects_dir=subjects_dir)
+
+full_label_names = []
+for full_label in full_labels:
+    if full_label.name.startswith('unknown'):
+        continue
+    full_label_names.append(full_label.name)
```

Comment:
or.. 

full_label_names = [full_label.name for full_label in full_labels
                                if full_label.name.find('unknown') == -1]

Reply:
Do you think it's that ugly? :D
Anyways, I used your suggestion.

Reply:
No, just makes for smaller code.